### PR TITLE
docs: add Argo CD playground to faq.md

### PR DESCRIPTION
**Pull request by balajisiva**
Updated with a link to a community Argo Hands-on playground setup.

Signed-off-by: Balaji Siva <bsivasub@gmail.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -224,4 +224,7 @@ resource.customizations.health.bitnami.com_SealedSecret: |
   hs.status = "Healthy"
   hs.message = "Controller doesn't report resource status"
   return hs
-```
\ No newline at end of file
+```
+## Is there a community Argo Hands-on Playground?
+
+Yes, you can try the [Argo Playground](https://www.opsmx.com/isd-platform/argo-sandbox/) for trying ArgoCD.  The playground already has target cluster and sample applications setup and you can explore features of ArgoCD with a Google login.
```
Comment (crenshaw-dev) [inline docs/faq.md:230]:
```suggestion
Yes, you can try the [Argo Playground](https://www.opsmx.com/isd-platform/argo-sandbox/) for trying Argo CD.  The playground already has target cluster and sample applications setup and you can explore features of Argo CD with a Google login.
```

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

**Comment by todaywasawesome** [comment]
I don't think we should link to vendor content like this without having a framework in place for how and when we'll allow it.

**Comment by balajisiva** [comment]
@todaywasawesome @crenshaw-dev This Argo Instance is a pure OSS Argo Instance, not a vendor version.  You can check it out if you would like.   It takes a google auth, and it connects to the Argo Instance.   Nothing vendor-specific.

We thought the community could benefit from a free OSS Instance to play with.   Prior hands-on lab from Katacoda was decommissioned, so we want to help increase Argo CD adoption.

**Comment by crenshaw-dev** [comment]
I'm inclined to accept, but I'm sympathetic to the need for some kind of guidelines.

Proposal:

> ### Links to vendor-maintained resources in official docs
> 
> Community resources maintained by Argo CD vendors are greatly appreciated!
> 
> In the interest of fairness, the following must be true before a resource is linked from official docs:
> 
> 1) The resource must be overwhelmingly (almost exclusively) "about" Argo CD. For example, a "how-to" guide with a vendor logo at the top is fine. The same guide interspersed with vendor ads is not acceptable.
> 2) Using the resource must not require providing any contact information to the vendor. For example, the user should not be required to send the vendor the name, phone, email, etc. before accessing the resource.

tbh, I'd love to have links to both this demo instance and CodeFresh's certificate course (which, as far as I can tell, is totally vendor-neutral besides logo placement). They're great resources that feel primarily "for" the community, while also benefitting the vendor with name recognition and just general good will.

To satisfy the above conditions, I think OpsMx would have to confirm that they don't collect the Google login information from the demo instance. (I assume that login is primarily to mitigate bot/spam/malicious access.)

I think the CodeFresh certification would need a no-sign-in option. Not sure that's something CodeFresh would be amenable to. But the "requirements" recommendation is certainly also up for discussion. :-)

**Comment by crenshaw-dev** [comment]
@todaywasawesome what do you think? Do those guidelines seem sufficient?

Probably something we should talk about at the next maintainers' meeting.

**Comment by wanghong230** [comment]
This will be quite controversial. I prefer to keep it neutral. So, I am against this.
Anyone can tell this is a promotion in one way or another. Please stay true.
We already have this vendor-neutral instance: https://cd.apps.argoproj.io/applications?showFavorites=false&proj=&sync=&health=&namespace=&cluster=&labels=

**Comment by crenshaw-dev** [comment]
If we want to go with absolutely "no vendor-site outbound links", then we should remove the existing ones from the [README](https://github.com/argoproj/argo-cd/blob/master/README.md), and we should document the policy.

But I'd be disappointed to see that helpful, vendor-neutral content leave our docs.

**Comment by hblixt** [comment]
Im not against linking to "free" resources hosted by vendors, but IMO those could have their own section in the README. We shouldnt have vendor resources listed as official project resources, unless they're managed/maintained by the project.  I like @crenshaw-dev  proposal, but maybe we can be a little more lenient on the second point? 

> > 2. Using the resource must not require providing any contact information to the vendor. For example, the user should not be required to send the vendor the name, phone, email, etc. before accessing the resource.

2. Vendors may require login for resource access, but no information may be collected, or used, for any other purpose than authentication without explicit permission from the user.

**Comment by wojtekidd** [comment]
There is one sandbox already as @wanghong230 mentioned. I know it's read-only, but perhaps one way to increase the adoption would be to make sure that the quickstart guides for every single Argo component are well-written and encourage folks to start their own Argo instances.

The other way around it would be to send someone to a 'vanilla' Argo instance rather than **a branded landing page from a company that monetizes Argo**:

Screenshot 2022-11-23 at 19 20 20

I can think of many different ways of actually doing this vendor-neutrally. We can chat about it during one of the Argo community meetings rather than just sneaking in a PR and hoping for the best 😉

**Comment by crenshaw-dev** [comment]
> There is no way to verify that. And even if this is true today, there is no guarantee that this will be true tomorrow.

I think the reputation damage caused by OpsMx using that data for marketing is sufficient deterrent. But I'm sympathetic to the argument that it might not be sufficient.

**Proposal:**

We add a new section to the bottom of the README. The new section looks like the example below. And it contains links to the Akuity- and CodeFresh-contributed resources which are currently linked in the README, as well as a _direct_ link to this demo instance (skipping the landing page).

> ### Vendor-sponsored community resources
> 
> The following links are to resources hosted by Argo CD vendors (companies which use Argo CD as part of their software offerings). These resources are vendor-neutral and do not require providing information for marketing purposes.
> 
> <links>

How's that feel to folks?

**Comment by hblixt** [comment]
> > There is no way to verify that. And even if this is true today, there is no guarantee that this will be true tomorrow.
> 
> I think the reputation damage caused by OpsMx using that data for marketing is sufficient deterrent. But I'm sympathetic to the argument that it might not be sufficient.
> 
> **Proposal:**
> 
> We add a new section to the bottom of the README. The new section looks like the example below. And it contains links to the Akuity- and CodeFresh-contributed resources which are currently linked in the README, as well as a _direct_ link to this demo instance (skipping the landing page).
> 
> > ### Vendor-sponsored community resources
> > The following links are to resources hosted by Argo CD vendors (companies which use Argo CD as part of their software offerings). These resources are vendor-neutral and do not require providing information for marketing purposes.
> 
> How's that feel to folks?

Im good with that.

**Comment by balajisiva** [comment]
@crenshaw-dev I am good with the proposal. I assume that you will do the new PR. or do you want me to do it?  I don't have links to other resources from other vendors you are referring to.  I assume we can add something like

Argo Demo  (maintained by OpsMx) 

or some variation of it so that vendor have some community visibility as a support of the community.

**Comment by crenshaw-dev** [comment]
PR: https://github.com/argoproj/argo-cd/pull/11511

@balajisiva personally, I feel better leaving the vendor name out of the link title. If the title seems like something that would be useful to the user, they'll click it and then see which vendor is hosting it. My feelings about this aren't super strong though.

The PR removes two Codefresh-hosted links. I haven't read those until now, and I would advocate for their removal regardless of whether we were adding a new vendor-hosted-resources section. If Codefresh folks (or anyone) disagrees with my assessment, I can further justify that position. I'd rather not; I think it's very, very clear.

I don't want to give the impression I'm picking on Codefresh. Their contributions to the project are [extensive](https://argo.devstats.cncf.io/d/4/company-statistics-by-repository-group?orgId=1&var-period=m&var-metric=activity&var-repogroup_name=argoproj%2Fargo-cd&var-companies=%22Codefresh%20Inc.%22) (even [not counting Pasha's work](https://argo.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=argoproj%2Fargo-cd&var-country_name=All&var-companies=All) - should probably fix attribution 😄). But I think the README should point to content that is overwhelmingly _about_ Argo CD. I don't think either of the links I removed meet that threshold.

**Comment by crenshaw-dev** [comment]
Honestly, the links should probably be cleaned up in general. I imagine a lot of the content is outdated anyway.

**Comment by balajisiva** [comment]
@crenshaw-dev I am good with proposal of leaving vendor names out of the link.   I will close this PR and open a new for the Readme.md based on your proposal. Thanks

**Pull request closed without merging**